rules/test.bzl

```diff
@@ -104,15 +104,12 @@ def _make_test(name, test_rule, **kwargs):
     Helper to create an individual test
     """
     runner = kwargs.pop("runner", None) or _DEFAULT_APPLE_TEST_RUNNER
-    test_attrs = {k: v for (k, v) in kwargs.items() if k not in _APPLE_BUNDLE_ATTRS}
+    test_attrs = {k: v for (k, v) in kwargs.items() if k in _IOS_TEST_KWARGS}
```

Comment:
This has been broken for a while it seems but afaik the right logic here is to grab the attrs for the unit test rule.. not what it was originally doing. Without this change `features` is dropped because it appars in `_APPLE_BUNDLE_ATTRS` (because every rule supports the `features` attr).

Reply:
My guess is this might have been to support custom rules with attributes unknown to this macro?

Reply:
Updated this maintain the original logic with the addition of including implicit rule attrs as well (like `features)